src/graphql/queries/lambdas/sendNotifications.graphql

```diff
@@ -0,0 +1,53 @@
+query GetUserDetails_Sn($id: ID!) {
```

Comment:
There has to be a better place for this file.

Even with it's own folder, we're mixing up queries to go in the client, with queries that go on the "server"